pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -671,7 +673,14 @@ stdenv.mkDerivation {
   preFixup = lib.optionalString withEfi ''
     mv $out/lib/systemd/boot/efi $out/dont-strip-me
   '';
-  postFixup = lib.optionalString withEfi ''
+
+  # Wrap in the correct path for LUKS2 tokens. Must be after the fixup phase
+  # or the rpath cleanup removes the directories again.
```

Comment:
Is this comment still up-to-date? I don't see any rpath being set here.

Reply:
No.